src/Surfnet/MessageBirdApiClientBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -57,7 +58,7 @@ public function testAuthorizationIsRequired()
     {
         $this->assertConfigurationIsInvalid(
             ['surfnet_message_bird_api_client' => []],
-            'child node "authorization"'
+            'child config "authorization"'
```

Comment:
Nitpick. I agree that your change made the error message more readable, however you did remove some meaning from the original mention that the authorization child `node` is invalid. 

Feel free to leave this as is. Just sayin'